client/src/lib/fetch.js

```diff
@@ -58,3 +68,63 @@ export async function createMeetup({
 
   return { res, resJSON: await res.json() };
 }
+
+export async function getMeetup(id) {
+  const getMeetupEndpoint = `${SERVER_URL}/meetup/${id}`;
+
+  const res = await fetch(getMeetupEndpoint, {
+    method: "GET",
+    credentials: "include",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
```

Comment:
We shouldn't need to specify GET or headers here.
```suggestion
  const res = await fetch(getMeetupEndpoint, {
    credentials: "include"
  });
```